.github/CONTRIBUTING.md

```diff
@@ -33,8 +33,10 @@ Due to the fact that this project is entirely volunteer-run (i.e. no one is paid
 to work on Python full-time), we unfortunately can make no guarantees as to if
 or when a core developer will get around to reviewing your pull request.
 If no core developer has done a review or responded to changes made because of a
-"changes requested" review, please feel free to email [python-dev](https://mail.python.org/mailman3/lists/python-dev.python.org/) to ask if
-someone could take a look at your pull request.
+"changes requested" review, please consider seeking assistance through the
+[Core Development Discourse category](https://discuss.python.org/c/core-dev/23).
+
+Please note that the traditional `python-dev` mailing list has been retired.
```

Comment:
Looking forwards, we don't need to note this (and I think you can't send new emails to that list, so it is sort-of redundant)
```suggestion
```